bciers/apps/reporting-e2e/project.json

```diff
@@ -2,6 +2,7 @@
   "name": "reporting-e2e",
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "apps/reporting-e2e/src",
-  "targets": {},
-  "implicitDependencies": ["reporting"]
+  "implicitDependencies": ["reporting"],
+  "// targets": "to see all targets run: nx show project reporting-e2e --web",
```

Comment:
what's the "// targets" for? is that some weird way of writing comments in a json-legal way?

Reply:
Those are something generated by Nx. I'll have to dig in and determine exactly what they are and if there's a better approach!